File: pkg/models/versions.go
package models

// Version is the model struct for Version
// swagger:model Version
type Version struct {
	ID               int64  `json:"id"`
	Version          string `json:"version"`
	AppID            string `json:"appId"`
	Disabled         bool   `json:"disabled"`
	DisabledMessage  string `json:"disabledMessage,omitempty"`
	NumOfClients     int64  `json:"numOfClients"`
	NumOfAppStartups int64  `json:"numOfAppStartups"`
}

File: ui/src/models/versions.js
export default class Version {
  constructor (json = {}) {
    this.id = json.id;
    this.versionNum = json.version;
    this.appId = json.appId;
    this.isDisabled = json.disabled;
    this.disabledMessage = json.disabledMessage;
    this.numOfAppLaunches = json.numOfAppLaunches;
    this.currentInstalls = json.numOfCurrentInstalls;
    this.lastLaunchedAt = json.lastLaunchedAt;
  }
}
